gravitee-apim-rest-api/docker/Dockerfile-dev

```diff
@@ -60,29 +58,3 @@ HEALTHCHECK --interval=5s --timeout=3s --retries=6 --start-period=30s \
             CMD curl --user admin:adminadmin --fail http://localhost:18083/_node/health || exit 1
 
 
-
-
-FROM eclipse-temurin:11-jre-focal
-LABEL maintainer="contact@graviteesource.com"
-
-ARG GRAVITEEIO_VERSION=0
-ENV GRAVITEEIO_HOME /opt/graviteeio-management-api
-
-ADD ./gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION}.tar.gz /tmp/
-
-RUN apt-get update \
-    && apt-get --yes upgrade \
-    && apt-get --yes install htop \
-    && mv /tmp/gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION} ${GRAVITEEIO_HOME} \
-    && rm -rf /tmp/* \
-    && rm -rf /var/lib/apt/lists/* \
-	&& chgrp -R 0 ${GRAVITEEIO_HOME} \
-    && chmod -R g=u ${GRAVITEEIO_HOME}
-
-WORKDIR ${GRAVITEEIO_HOME}
-EXPOSE 8083
-ENTRYPOINT ["./bin/gravitee"]
-VOLUME ${GRAVITEEIO_HOME}/logs
-CMD ["./bin/gravitee"]
-HEALTHCHECK --interval=5s --timeout=3s --retries=6 --start-period=30s \
-            CMD curl --user admin:adminadmin --fail http://localhost:18083/_node/health || exit 1
```

Comment:
It's a legacy thing that should have been removed earlier?

Reply:
Yes, it should be part of another PR on master. It comes from the merge of the "performance branch"

Reply:
https://github.com/gravitee-io/gravitee-api-management/pull/1073